openmrs-19/src/main/java/org/motechproject/openmrs19/service/OpenMRSConceptService.java

```diff
@@ -11,55 +11,134 @@
 public interface OpenMRSConceptService {
 
     /**
-     * Returns the UUID of the concept with the given {@code name}.
+     * Returns the UUID of the concept with the given {@code name}. Configuration with the given {@code configName} will
```

Comment:
I think that methods without configName in OpenMRS services are redundant. Can you remove them ? The only reason to use them is to have backward compatibility, however after huge refactor in OpenMRS it have been already lost.